tools/generate-gemm-test.py

```diff
@@ -537,8 +613,10 @@ def split_ukernel_name(name):
               .extended_weights(true)
             .mr(mr).nr(nr).kr(kr).sr(sr).m(mr).n(nr).ks(3)
             .a_offset(NextPrime(mr * k_block * 3 + 1))
-            $if KERNELTYPE == 'qc4w':
+            $if KERNELTYPE in ['qb4w', 'qc4w']:
               .b_zero_point(8)
+            $if KERNELTYPE in ['qb4w']:
+              .bl(kr * sr * 2)
         , test_func, isa_check)
         .loop_k(1, k_block * 3, k_block + 1)
```

Comment:
We might have to remove some of these tests when we introduce KR > 1 kernels because these may violate bl constraints. But I think this is OK for now for the scalar kernels.